HW_1/task_1.py

```diff
@@ -0,0 +1,17 @@
+# Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. 
+# Дано a, b, c — стороны предполагаемого треугольника. Требуется сравнить длину каждого
+# отрезка-стороны с суммой двух других. Если хотя бы в одном случае отрезок окажется больше
+# суммы двух других, то треугольника с такими сторонами не существует. Отдельно сообщить
+# является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+
+a, b, c = map(float, input("Введите три стороны прямоугольника (через пробел) --> ").split())
+
+if a > b + c or b > a + c or c > a + b:
+    print("Не может существовать треугольника с такими сторонами")
+elif a == b == c:
+    print("Данный треугольник равносторонний")
+elif a == b != c or a == c != b or b == c != a:
+    print("Данный треугольник равнобедренный")
+else:
+    print("Данный треугольник разносторонний")
```

Comment:
Разрешить я внесу несколько правок.
```suggestion
elif a == b or a == c or b == c:
    print("Данный треугольник равнобедренный")
else:
    print("Данный треугольник разносторонний")
```
или так
```suggestion
elif a != b != c:
    print("Данный треугольник разносторонний")
else:
    print("Данный треугольник равнобедренный")
```
Внимательно разберите мои правки.